nectar/config.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+from builtins import object
 
 import os
 import tempfile
 
 import requests.auth
 
+temp_files = []
```

Comment:
Maybe I can revert this file and make some change in tests. Maybe mentioned exception comes from bad test and not from bad code in this file.

But yes, you know this code better so I will wait for your decision.